lib/api/controllers/authController.js

```diff
@@ -7,26 +7,46 @@ module.exports = function AuthController (kuzzle) {
 
   this.passport = new PassportWrapper(kuzzle);
 
-  this.login = function (requestObject) {
+  this.logout = function (requestObject, context) {
+    var
+      deferred = q.defer(),
+      response;
```

Comment:
Nitpicking: should be moved inside the appropriate `then()` callback, to restrict variable scoping.